hedera-node/configuration/previewnet/application.properties

```diff
@@ -7,6 +7,7 @@
 bootstrap.genesisPublicKey=c249a323c878f5b5e2daccda6d731e6fdc32f870228d1cd4fae559d947dbc36c
 contracts.chainId=297
 contracts.systemContract.updateNFTsMetadata.enabled=true
+contracts.evm.ethTransaction.zeroHapiFees.enabled=true
```

Comment:
you might want to check to see if this is the appropriate place to set this to true if your intention is to only turn this on for local node.

Reply:
I think this was supposed to be the dev file so that it's only on for local node and not yet on for any public env